packages/internals/src/wasm.ts

```diff
@@ -1,14 +1,14 @@
-import prismaFmt from '@prisma/prisma-fmt-wasm'
+import prismaSchemaWasm from '@prisma/prisma-schema-wasm'
 
 import { WasmPanicRegistry } from './WasmPanicRegistry'
 
 // Note: using `import { dependencies } from '../package.json'` here would break esbuild with seemingly unrelated errors.
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { dependencies } = require('../package.json')
 
-export { prismaFmt }
+export { prismaSchemaWasm }
```

Comment:
Nit: do we really need a named export in a `wasm.ts` file to have a `-Wasm` postfix?

Reply:
`prismaSchema` is usually used in the codebase to refer to the Prisma schema file, so I think we can be very explicit here

Reply:
I agree, just `prismaSchema` would be dangerous.